lib/testers/rule-tester.js

```diff
@@ -252,7 +253,8 @@ class RuleTester {
         const testerConfig = this.testerConfig,
             requiredScenarios = ["valid", "invalid"],
             scenarioErrors = [],
-            result = {};
+            result = {},
+            eslint = this.linter;
```

Comment:
Is there a reason to create a different variable here? If so, can we call it `linter` rather than `eslint`?